docs.go

```diff
@@ -93,6 +98,12 @@ Workflow options:
     -g --graph
       Show links on graph pages.
 
+    --stacked
+      Output single link for the stacked graph of selected data.
+
+    --normal
```

Comment:
I'm not sure, it's intended to use inside aliases and there is shortage for short keys, which may be used for something more frequent.